schema/snapcraft.yaml

```diff
@@ -57,16 +57,6 @@ properties:
     uniqueItems: true
     items:
       - type: string
-  license:
-    type: string
-    description: path to a license file.
-  license-agreement:
-    type: string
-    description: action to take on licensing.
-    enum:
-      - explicit
-  license-version:
-    description: license version (used to re-trigger an agreement action).
```

Comment:
On the bright side, once we can deliver as a snap we can jump major version numbers without worrying about SRUs.